<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAddEditMontajes</class>
 <widget class="QDialog" name="frmAddEditMontajes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>464</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Montaje de unidades</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img16/gr-lida.png</normaloff>:/img16/gr-lida.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,1,0">
   <item>
    <widget class="QLabel" name="lb_Montaje_1">
     <property name="text">
      <string>Seccionar la unidad, carpeta o CD-Rom que desea montar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lb_Montaje_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Tipo de montaje:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxMontaje_type_drive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDirFile">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/carpeta_0.png</normaloff>:/img16/carpeta_0.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="txtMontaje_path">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnDeleteIso">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnSubirIso">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/go-up.png</normaloff>:/img16/go-up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QListWidget" name="lw_MultiIso">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDirFile_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnBajarIso">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>btnDeleteIso</zorder>
     <zorder>btnSubirIso</zorder>
     <zorder>btnDirFile_clear</zorder>
     <zorder>btnDirFile</zorder>
     <zorder>btnBajarIso</zorder>
     <zorder>lw_MultiIso</zorder>
     <zorder>txtMontaje_path</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_Montaje_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Letra a montar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxMontaje_letter">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb_Montaje_4">
       <property name="text">
        <string>Index CD-Rom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxMontaje_cdrom">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lb_Montaje_5">
       <property name="text">
        <string>Etiqueta para la unidad es opcional:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtMontaje_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lb_Montaje_6">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Modo CD-Rom:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbxMontaje_mode_cdrom">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lb_Montaje_7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Otras opciones de montajes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxMontaje_opt_mount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="chkMontaje_opt_freesize">
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>4</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lb_Montaje_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Opci√≥n avanzada: [sectorsbytesize, sectorsperhead, heads, cylinders]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="txtMontaje_opt_size_mount">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnMontaje_opt_size_mount_clear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QSlider" name="h_SliderMontaje_freesize">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>4000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lb_Montaje_info">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxMontaje_type_drive</tabstop>
  <tabstop>btnDirFile</tabstop>
  <tabstop>txtMontaje_path</tabstop>
  <tabstop>lw_MultiIso</tabstop>
  <tabstop>btnDirFile_clear</tabstop>
  <tabstop>btnSubirIso</tabstop>
  <tabstop>btnBajarIso</tabstop>
  <tabstop>btnDeleteIso</tabstop>
  <tabstop>cbxMontaje_letter</tabstop>
  <tabstop>cbxMontaje_cdrom</tabstop>
  <tabstop>txtMontaje_label</tabstop>
  <tabstop>cbxMontaje_mode_cdrom</tabstop>
  <tabstop>cbxMontaje_opt_mount</tabstop>
  <tabstop>chkMontaje_opt_freesize</tabstop>
  <tabstop>h_SliderMontaje_freesize</tabstop>
  <tabstop>txtMontaje_opt_size_mount</tabstop>
  <tabstop>btnMontaje_opt_size_mount_clear</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
